mod/ping.php

```diff
@@ -123,11 +123,10 @@ function ping_init(&$a) {
 		$mail = count($mails);
 
 		if ($a->config['register_policy'] == REGISTER_APPROVE && is_site_admin()){
-			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created`, COUNT(*) as `total` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
-			if ($regs)
-				$register = $regs[0]['total'];
+			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
+			$register = count($regs);
```

Comment:
We are having a function that checks the return data from a query:

````
if (dbm::is_result($regs))
````
Although in this case it is not really needed, I would like to have it added for consistency reasons.

Could you add it?